pungi/run.py

```diff
@@ -129,7 +129,7 @@ def run_snakemake_workflow(config_path, snake_file_path, output_dir_path, conda_
 
     snakemake_args = ['snakemake', '--snakefile', snake_file_path, '--configfile', config_path, '--directory',
                       output_dir_path, '--conda-prefix', conda_env_dir_path, '--conda-frontend',
-                      'mamba', '--use-conda', '--reason', '--rerun-incomplete', '--printshellcmds']
+                      'mamba', '--software-deployment-method conda', '--rerun-incomplete', '--printshellcmds']
```

Comment:
Now that the [libmamba solver is the default solver for conda](https://conda.org/blog/2023-11-06-conda-23-10-0-release/), I wonder if we might be able to drop the `--conda-frontend mamba` argument? Normal conda should also be pretty fast now. @LeeBergstrand thoughts?

Reply:
```
--conda-frontend
Possible choices: conda, mamba

Choose the conda frontend for installing environments. Mamba is much faster and highly recommended.

Default: “mamba”
```

@jmtsuji Snakemake 8 still lists mamba as the default, so I would leave it be. Mamba is still faster a little bit faster than Conda because all the other code is written in C++ where as in Conda it is still written in python other than mamba imported library for figuring out the package dependancies.

Reply:
Sounds good -- if mamba is still the snakemake 8 default, then let's leave it as-is like you suggested 👍